hazelcast-oss/get-hz-all-url.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
You can add some comments here about the reasoning behind this script. Mention for sure that:
- we need to extract the version manually (not with Maven), since `mvn` doesn't work with `dockerx` emulation
- how you extract the snapshot value (that's it's always the first entry in the XML file)

Reply:
Sure, I will add comments to the script.